src/math/Vector4.js

```diff
@@ -1060,4 +1051,6 @@ class Vector4 {
 
 }
 
+Vector4.prototype.isVector4 = true;
```

Comment:
This breaks tree-shaking, afaik. That's why we don't use this style anymore.

Reply:
Gotcha, I was wondering why we were assigning at each instance creation. Reverted in https://github.com/mrdoob/three.js/pull/30779/commits/6ef4117e54c6e4a33cfce60d744c028c43702890